api/client/samples/prevented_plant/prevented_plant_xgboost.ipynb

```diff
@@ -34,6 +34,60 @@
    "source": [
```

Comment:
<p>"For Windows users, geopandas is best installed by using anaconda and the command `conda install geopandas'. Please see <a href="http://geopandas.org/install.html" target="_blank">http://geopandas.org/install.html</a> for further details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Does conda install also install shapely?

Reply:
updated instruction